runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java

```diff
@@ -173,7 +147,8 @@ public void onClanMemberJoined(ClanMemberJoined event)
 
 			for (final Player player : client.getPlayers())
 			{
-				if (player != null && memberName.equals(Text.toJagexName(player.getName())))
+				if (player != null && memberName.equals(Text.toJagexName(player.getName())) &&
```

Comment:
you should instead compare localPlayer name to memberName before loop

Reply:
Done!